### PR TITLE
Bump iptables image for ztunnel

### PR DESCRIPTION
**Pull request by yankay**
**Please provide a description of this PR:**

Update base distroless image for ztunnel since there are fixed CVEs

### CODE REVIEW

**Review by ericvn — CHANGES_REQUESTED** [review]
I believe this was made part of the bin/update_deps.sh script, so would you please run that and include all the updates. Thanks.

File: pilot/docker/Dockerfile.ztunnel [hunk 1]
```diff
@@ -12,8 +12,8 @@ FROM ${ISTIO_BASE_REGISTRY}/base:${BASE_VERSION} as debug
 # This image is a custom built debian11 distroless image with multiarchitecture support.
 # It is built on the base distroless image, with iptables binary and libraries added
 # The source can be found at https://github.com/istio/distroless/tree/iptables
-# This version is from commit 6faf2f9348d382c6a95bbc3171b89406103cb0ad.
-FROM ${ISTIO_BASE_REGISTRY}/iptables@sha256:64ba191166d2c3f87815148cc1a9f530b770e0dfe81acb164ab79cbb31da582c as distroless
+# This version is from commit a8b3fb577adb785211ce704fdf892983fc268b11.
```
Comment (ericvn) [inline pilot/docker/Dockerfile.ztunnel:15]: Remove the comment line since the script won't be updating that.

**Review by howardjohn — APPROVED** [review]

**Comment by ericvn** [comment]
Note that a later PR updating update_deps and running it supersedes this PR.

**Comment by yankay** [comment]
It has been update by https://github.com/istio/istio/pull/44160
/close

**Pull request closed without merging**